### DISCUSSION TITLE
Core Features

### DISCUSSION THREAD

**Discussion by k9withabone** [audiobookshelf-flutter/audiobookshelf_flutter#1] [Ideas]
This is a continuation of [the discussion](https://github.com/advplyr/audiobookshelf-app/discussions/428) in the audiobookshelf-app repository. 

What core features should we focus on for the initial release?

**Comment by k9withabone** [comment]
Here is a basic list of things I have thought of so far:

- Login and logout
- Library list
- Library home page
- Basic library page (no filtering or sorting yet)
- Basic series page (no filtering or sorting yet)
- Collections page
- Authors page
- Stream items
  - Sync position with server

**Comment by Hackmodford** [comment]
I've created a PR that's get's us most of the way there thanks to the Audiobookly app.
https://github.com/audiobookshelf-flutter/audiobookshelf_flutter/pull/3

**Reply by Hackmodford** [reply]
merged this in

**Comment by Hackmodford** [comment]
I think the first order of business is to get the api repo setup and publishing to pub.dev, then we can modify the app to use that instead of embedding the sub project.

**Reply by k9withabone** [reply]
Agreed, although we might want to wait to publish it until it is in a more complete state. I've been working on the API documentation so we have something to match it against.

**Reply by k9withabone** [reply]
I'm going to start working on the API repo. The first thing I'm going to do is copy the `lib` folder over and clean it up a little bit. Then I'll work on adding everything on the [API docs](https://api.audiobookshelf.org).
